### PR TITLE
feat(fsl) : new app

### PR DESCRIPTION
**Pull request by floriansipp**
Once [#43](https://github.com/CHORUS-TRE/images/pull/43) is validated

### CODE REVIEW

File: apps/fsl/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+ARG DEBIAN_FRONTEND=noninteractive
+RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
+    --mount=type=cache,target=/var/lib/apt,sharing=locked \
+    apt-get update -qy && \
+    apt-get install --no-install-recommends -qy \
+    ca-certificates curl dc file libgomp1 libquadmath0 locales python3 && \
+    locale-gen en_US.UTF-8 en_GB.UTF-8 && \
+    curl -sSL https://fsl.fmrib.ox.ac.uk/fsldownloads/fslinstaller.py -o fslinstaller.py && \
+    sed -i -E "s/(printmsg\(([^,]+, )?end='(\\\\r)?')/# SILENE \\1/g" ./fslinstaller.py && \
```
Comment (greut) [inline apps/fsl/Dockerfile:18]: Well, that one was missing :)

```suggestion
    sed -i -E "s/(printmsg\(([^,]+, )?end='(\\\\r)?')/# SILENCE \\1/g" ./fslinstaller.py && \
```

File: apps/fsl/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    python3 ./fslinstaller.py \
+        -d /usr/local/fsl \
```
Comment (greut) [inline apps/fsl/Dockerfile:20]: Why not `/apps/${APP_NAME}`?

File: apps/fsl/Dockerfile (hunk 1 above)
Comment (floriansipp) [reply apps/fsl/Dockerfile:18]: Nice catch, i used the hip dockerfile, we'll need to fix this there too

File: apps/fsl/Dockerfile (hunk 2 above)
Comment (floriansipp) [reply apps/fsl/Dockerfile:20]: hum from memory i'd say that it has some dependencies that are better handled if installed the way it is done there. We can try to install it in /apps/${APP_NAME} but only once i can run some scripts to see that everything works as intended

Comment (greut) [reply apps/fsl/Dockerfile:20]: it holds the config files, so I guess it's ok. Ignore my comment.

File: apps/fsl/build.sh [hunk 3]
```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
```
Comment (greut) [inline apps/fsl/build.sh:2]:
```suggestion

set -e

```

Comment (greut) [reply apps/fsl/build.sh:2]: otherwise, failing to build the image won't produce a failure.

Comment (floriansipp) [reply apps/fsl/build.sh:2]: Indeed, i need to put that by default for every image, i'll remember it at one point ^^'

**Review by greut — APPROVED** [review]

**Pull request merged**